src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -395,15 +397,15 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 		this.container.style.height = '100%';
 
 		this.columns = this.resultSet.columnInfo.map((c, i) => {
-			let isLinked = c.isXml || c.isJson;
+			let isLinked = c.isXml;
```

Comment:
Might as well just remove this and use c.isXml directly below

(something to consider would be to have the queryResultTextFormatter always be used and just check in there whether the column is XML, but no big preference on my end)

Reply:
the isXml information is available in the columnInfo but not the slickgrid's columnDefinition, If I always use the new formatter, the isXml information needs to be retrieved from the resultSet.columnInfo again. so I am going to leave it as is.